tests/tck/features/match/MultiQueryParts.feature

```diff
@@ -230,8 +333,22 @@ Feature: Multi Query Parts
     Then the result should be, in order:
       | scount |
       | 270    |
+    # And the execution plan should be:
+    # | id | name           | dependencies | operator info                                              |
+    # | 12 | Aggregate      | 13           |                                                            |
+    # | 13 | BiInnerJoin    | 15, 11       |                                                            |
+    # | 15 | Project        | 5            |                                                            |
+    # | 5  | AppendVertices | 4            | {  "props": "[]" }                                         |
+    # | 4  | Traverse       | 3            | { "vertexProps": "" }                                      |
+    # | 3  | Traverse       | 14           | {  "vertexProps": "[{\"props\":[\"name\"],\"tagId\":2}]" } |
+    # | 14 | IndexScan      | 2            |                                                            |
+    # | 2  | Start          |              |                                                            |
+    # | 11 | Project        | 10           |                                                            |
+    # | 10 | AppendVertices | 9            | {  "props": "[]" }                                         |
+    # | 9  | Traverse       | 8            | {  "vertexProps": "[{\"props\":[\"name\"],\"tagId\":2}]" } |
+    # | 8  | Argument       |              |                                                            |
     # Below scenario is not suppoted for the execution plan has a scan.
-    When executing query:
+    When profiling query:
```

Comment:
Why change this?

Reply:
I was going to add a execution plan check here